piksi_gps/piksi_gps/piksi_gps_pub.py

```diff
@@ -16,16 +16,13 @@
 # Node Name: gps
 # Topic: /gps/fix
 
-
-
 class GPSPublisher(Node):
-    """ A template node containing a publisher, subscriber, and a parameter. 
-		Calculates the sum of an input and some fixed integer
-
+    """ 
+    A ROS node that publishes GPS, magnometer and accelerometer data
```

Comment:
More detail preferable in the form of an extended summary (see [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#extended-summary)). This is a node specifically for the piksi multi and the summary should reflect this and any other strict application details of the node.